scripts/bootstrap_db_for_local_tests.py

```diff
@@ -16,18 +16,31 @@
 
 
 def create_workspace_resource(resource_id: str):
```

Comment:
this file if for doing manual testing?

Reply:
Yes, to create a few records, if you don't want to manually sit and do posts... added it to the scripts directory so it's outside the API